.github/workflows/podvm_binaries.yaml

```diff
@@ -0,0 +1,65 @@
+name: Create Pod VM Binaries Image
+on:
+  workflow_run:
+    workflows: ["Create Pod VM Builder Image"]
+    types:
+      - completed
+
+jobs:
+  build:
+    name: Create pod vm binaries image
+    if: ${{ github.event_name == 'workflow_dispatch' || github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [centos, ubuntu]
+        arch: [amd64, s390x]
+        include:
+          - os: centos
+            dockerfile: Dockerfile.podvm_binaries.centos
+          - os: ubuntu
+            dockerfile: Dockerfile.podvm_binaries
+        exclude:
+          - os: centos
+            arch: s390x
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3
+
+    #- name: Set up QEMU
+    #  uses: docker/setup-qemu-action@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Login to Docker container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: docker.io
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+
+    - name: Login to Quay container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+
+    - name: Build and push
+      uses: docker/build-push-action@v3
+      with:
+        tags: |
+          quay.io/confidential-containers/podvm-binaries-${{ matrix.os }}-${{ matrix.arch }}:latest
+          quay.io/confidential-containers/podvm-binaries-${{ matrix.os }}-${{ matrix.arch }}:${{ github.sha }}
+        push: true
+        context: podvm
+        platforms: linux/amd64
+        file: |
+          podvm/${{ matrix.dockerfile }}
+        build-args: |
+          "CLOUD_PROVIDER=${{ matrix.provider }}"
```

Comment:
This is just a question. Where is `matrix.provider` is defined, and where is `CLOUD_PROVIDER` used?

Reply:
@yoheiueda good catch. This is a left over and is not used during the gh-action execution.